nservicebus/sagas/index.md

```diff
@@ -22,6 +22,24 @@ Design processes with more than one remote call to use sagas.
 While it may seem excessive at first, the business implications of the system getting out of sync with the other systems it interacts with can be substantial. It's not just about exceptions that end up in the logs.
 
 
+## No saga instance available
+
+If a message that doesn't start a saga is received but no related saga instance is found, then by default that message is discarded, i.e. it is possible that a saga instance had been already completed when further messages arrive and such messages should be discarded.
+
+NOTE: Always assume that messages can be delivered out of order, i.e. due to invoking the error recovery mechanism, network latency, and/or concurrent message processing.
+
+A similar message discarding behavior can also be invoked in case of saga instance creation. When a message is received that expects a saga instance to be there but that message that would create the saga instance has not yet been received or completed processing.
```

Comment:
I've reviewed content above, I hope I kept the essence of what you're saying, but starting from this place I'm not sure if I understand the intention.

E.g. I'm a bit confused about the fact that you introduce the discarding of a message here, separately. Isn't it the same behavior as above, only a different scenario/reason? If so, that should be moved to the paragraph before the "NOTE:" part.

Reply:
Above the note, I explain the default discard behavior. Below the note a scenario where you don't want to have that behavior due to out of order processing which is often not thought of.